src/typechecker/FStar.TypeChecker.Normalize.fst

```diff
@@ -999,10 +999,11 @@ let should_unfold cfg should_reify fv qninfo : should_unfold_res =
        && !plugin_unfold_warn_ctr > 0                   // and we haven't raised too many warnings
     then begin
       // then warn about it
+      let msg = BU.format1 "Unfolding name which is marked as a plugin: %s"
+                                    (Print.fv_to_string fv) in
+      BU.print1 "%s\n" msg;
```

Comment:
Remove degugging. Although sometimes I find that F* doesn't report this warning at all, or reports it too late after a very long and expensive tactic evaluation